docs/en/operation/Journal.md

```diff
@@ -109,15 +109,20 @@ and the master rpc port (Default:`19998`).
 
 ### Advanced configuration
 
+* `alluxio.master.embedded.journal.catchup.retry.wait`: Time for embedded journal leader to wait before retrying a catch up.
+ This is added to avoid excessive retries when server is not ready. Default: `1s`.
+* `alluxio.master.embedded.journal.entry.size.max`: The maximum single journal entry size allowed to be flushed.
+This value should be smaller than 30MB. Set to a larger value to allow bigger journal entry from using alluxio Catalog service. Default: `10MB`.
```

Comment:
Set to a larger value to allow larger journal entries when using the Alluxio Catalog service.